hedera-node/hedera-network-admin-service-impl/src/test/java/com/hedera/node/app/service/networkadmin/impl/test/handlers/NetworkAdminHandlerTestBase.java

```diff
@@ -340,7 +341,7 @@ protected void givenValidAccount(
                 isDeleted,
                 1_234L,
                 1_234_568L,
-                0,
+                new OneOf<>(Account.StakedIdOneOfType.UNSET, null),
```

Comment:
same here

Reply:
You can't use null here. The generated Account.java has this:
public Account {
        if (stakedId == null) {
        	throw new NullPointerException("Parameter 'stakedId' must be supplied and can not be null");
        }
    }

Reply:
Set static final OneOf<Account.StakedIdOneOfType> UNSET_STAKED_ID =
            new OneOf<>(Account.StakedIdOneOfType.UNSET, null); in their own modules and use UNSET_STAKED_ID in the code.